source/scripts/notesEditor.js

```diff
@@ -128,21 +132,24 @@ async function addNotesToDocument(note, editable) {
     let title = document.querySelector('#notes-title');
     let lastModified = document.querySelector('#notes-last-modified')
     let content = document.querySelector('#notes-content-input');
-
+    //empty the html items
     // populate html with notes data
     title.innerHTML = `<label for="title-input">Title:</label>
         <input type="text" id="title-input" name="title-input">
         `;
     let titleInput = document.querySelector('#title-input');
     titleInput.value = note.title;
-    lastModified.innerHTML += `${note.lastModified}`;
-    content.value += `${note.content}`;
-
-    // disable editing priveleges if in view only mode
+    lastModified.innerHTML = `${note.lastModified}`;
+    content.value = `${note.content}`;
+    // disable editing pages if in view only mode
     if (!editable) {
         //make input field uneditable
         titleInput.setAttribute('disabled', 'true');
         //make content textarea uneditable
         content.setAttribute('readonly', 'readonly');
     } 
+    else{
```

Comment:
This and the following lines are unnecessary code because by we add the notes to the page again every time we call this method and by default the inputs don't have the disabled and readonly attribute.